.maestro/create-offence.yaml

```diff
@@ -0,0 +1,69 @@
+appId: com.bratislava.enforcement
+---
+- launchApp
+- waitForAnimationToEnd
+# Set location to Dunajska street
+- setLocation:
+    latitude: 48.150636
+    longitude: 17.111754
+- assertVisible:
+    id: municipal-police
+- tapOn:
+    id: municipal-police
+- tapOn: 'Evidenčné číslo vozidla'
+- inputText: BT163CG
+- hideKeyboard
+- tapOn:
+    id: licencePlateContinue
+- runFlow:
+    when:
+      visible:
+        id: PAAS_PARKING_VIOLATION_DUPLICITY
+    commands:
+      - tapOn:
+          id: licencePlateContinue
+- extendedWaitUntil:
+    visible: 'Vyberte druh priestupku'
+    timeout: 10000
+- tapOn: 'Vyberte druh priestupku'
+- tapOn: 'Nerešpektovanie DZ'
+- tapOn: Pokračovať
+- runFlow:
+    when:
+      visible: Duplicita
+    commands:
+      - tapOn: Vytvoriť hlásenie
```

Comment:
Is it okay to not have `''` around multiple words string? In other plases, you use `''`, e.g. `- tapOn: 'Nerešpektovanie DZ'`

Reply:
no need to use `''`. I will remove all the occurrences